modules/302-vertical-pod-autoscaler/images/admission-controller/Dockerfile

```diff
@@ -1,8 +1,13 @@
-ARG BASE_ALPINE
-FROM registry.k8s.io/autoscaling/vpa-admission-controller:0.12.0@sha256:997b6efac7bda2b0fd9fe8b54feaeac5fb77cadbbe01a46806f3b0c84a354429 as artifact
-
-FROM $BASE_ALPINE
-COPY --from=artifact /admission-controller /
+ARG BASE_DISTROLESS
+ARG BASE_GOLANG_19_ALPINE
+FROM $BASE_GOLANG_19_ALPINE as artifact
+WORKDIR /src/
+RUN apk add --no-cache git mercurial && \
+    wget https://codeload.github.com/kubernetes/autoscaler/tar.gz/vertical-pod-autoscaler/v0.14.0 -O - | tar -xz --strip-components=1 -C /src/ && \
```

Comment:
Why do we need mercurial here ?

Reply:
Also please use git clone instead of wget

Reply:
Also please use werf format

Reply:
done